src/plugins/data/public/query/query_string/dataset_service/lib/index_type.ts

```diff
@@ -20,7 +20,9 @@ export const DELIMITER = '::';
 
 export const indexTypeConfig: DatasetTypeConfig = {
   id: DEFAULT_DATA.SET_TYPES.INDEX,
-  title: 'Indexes',
+  title: i18n.translate('data.datasetType.indexes', {
```

Comment:
is this the right key? i think the ci now verifies for us (i think miki set it up) but can you verify this doesn't fail the i18n generation script?